psen_scan_v2_tutorials/config/scanner2_config.yaml

```
@@ -1,7 +1,13 @@
 # %Tag(FULLTEXT)%
-sensor_ip: "192.168.0.20"
-host_ip: "192.168.0.50"
-host_udp_port_control: 50505 
-host_udp_port_data: 50506 
-prefix: laser_2
+sensor_ip: "192.168.0.20"    # IP address of the laser scanner
+prefix: laser_2              # This name is prefixed to the TF, where the scanner is mounted
+host_udp_port_control: 50505 # Make sure that not the same ports are used as in the first scanner config.
+host_udp_port_data: 50506
+## The following parameters are optional
+angle_start: -2.399          # The angle of the first measurement (-2.399 rad ~= -137.5 deg)
+angle_end: 2.399             # The angle of the last measurement (2.399 rad ~= 137.5 deg)
+intensities: false           # Receive intensity values
+resolution: 0.0017           # Scan resolution (0.0017 rad ~= 0.1 deg)
```

```suggestion
resolution: deg(0.1)         # Scan resolution (0.0017 rad ~= 0.1 deg)
```
Do python expressions work as described in http://wiki.ros.org/rosparam ?

---

If this works, update pilzde/psen_scan_v2/README.md, too

---

Yes, but these are special rosparam converters, not python ...